grails-app/domain/org/pih/warehouse/core/Organization.groovy

```diff
@@ -37,10 +37,10 @@ class Organization extends Party {
     }
 
     static constraints = {
-        code(nullable: false, unique: true,
+        code(nullable: false, blank: false, unique: true,
                 minSize: Holders.grailsApplication.config.openboxes.identifier.organization.minSize,
                 maxSize: Holders.grailsApplication.config.openboxes.identifier.organization.maxSize)
-        name(nullable: false, maxSize: 255)
+        name(nullable: false, blank: false, maxSize: 255)
```

Comment:
I assume whenever we make such a change, we should always align GORM constraints with DB constraints, hence a migration that adds this constraint should be added.
I'm not sure how the ddl would act in this case, because we do not have anything like "NOT EMPTY" in SQL and should probably do something like:
```sql
CHECK (name <> '')
```

Reply:
yep but we should keep in mind that CHECK is not available for `MySQL 5.7` and need to make sure that we are going to be using either version 8.^ or MariaDB